code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -355,8 +370,8 @@
 			S.start()
 		qdel(B)
 
-/datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
-	name = "!(%@ ERROR )!@$"
+datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
+	name = "Odd Black Liquid"
```

Comment:
Don't do the "\improper" we need to deprecate that.
it causes an odd Umlaut'd Y to appear in texts and UIs.

simply lowercase the name.